cmd/coap/main.go

```diff
@@ -32,33 +32,33 @@ import (
 
 const (
 	defPort              = "5683"
+	defProtocol          = "udp"
 	defNatsURL           = "nats://localhost:4222"
 	defLogLevel          = "error"
 	defClientTLS         = "false"
 	defCACerts           = ""
-	defPingPeriod        = "12"
 	defJaegerURL         = ""
 	defThingsAuthURL     = "localhost:8181"
 	defThingsAuthTimeout = "1s"
 
 	envPort              = "MF_COAP_ADAPTER_PORT"
+	envProtocol          = "MF_COAP_ADAPTER_PROTOCOL"
 	envNatsURL           = "MF_NATS_URL"
 	envLogLevel          = "MF_COAP_ADAPTER_LOG_LEVEL"
 	envClientTLS         = "MF_COAP_ADAPTER_CLIENT_TLS"
 	envCACerts           = "MF_COAP_ADAPTER_CA_CERTS"
-	envPingPeriod        = "MF_COAP_ADAPTER_PING_PERIOD"
```

Comment:
Don't forget to update this values in .env. Just curious, how many protocols are available for `MF_COAP_ADAPTER_PROTOCOL`?

Reply:
Protocols TCP and UDP. Let's keep only UDP at the moment, and add TCP if needed. I'll remove this env var.